src/routes/settings/Encrypt.tsx

```diff
@@ -39,6 +39,7 @@ export default function Encrypt() {
                 confirmPassword: ""
             },
             validate: (values) => {
+                setError();
                 const errors: Record<string, string> = {};
                 if (values.password !== values.confirmPassword) {
```

Comment:
could also add the check here that old != new password

Reply:
I had that initially but with your mutiny-node pr I thought it might be error code overload, what do you think?

Reply:
I had that initally but I though that might be error code overload, what do you think?

Reply:
May as well defend on both ends 🤷‍♂️

Reply:
nvm I see what you mean, validate runs before handleEncrypt so no double errors